bee-message/tests/message.rs

```diff
@@ -123,75 +106,6 @@ fn unpack_invalid_payload_length() {
     ));
 }
 
-#[test]
-fn invalid_no_strong_parents() {
```

Comment:
Shouldn't we still have a similar test ?

Reply:
Missed that one! All done.